Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -301,9 +320,24 @@ its :php:`__invoke()` method will be called:
 
 Read :ref:`how to configure dependency injection in extensions <dependency-injection-in-extensions>`.
 
-Existing event listeners can be overridden by custom implementations. For example,
-:file:`EXT:some_extension/Configuration/Services.yaml` could listen on the event
-:php:`\TYPO3\CMS\Frontend\Event\ModifyHrefLangTagsEvent`:
+..  index:: Override Event Listener; Event override
+..  _EventListenerOverride:
+
+Overriding event listeners
+--------------------------
+
+Existing event listeners can be overridden by custom implementations. This can be
+performed with both methods, using the PHP :php:`#[AsEventListener}` attribute or
+via :file:`EXT:some_extension/Configuration/Services.yaml`.
+
+For example, if a third-party extension listens on the event
+:php:`\TYPO3\CMS\Frontend\Event\ModifyHrefLangTagsEvent` with the following code:
+
+..  literalinclude:: _ServicesOverrideBase.php
+    :language: php
+    :caption: EXT:some_extension/Classes/EventListener/SeoEvent.php
+
+or via :file:`Services.yaml` legacy declaration:
```

Comment:
Please add an admonition:

When overriding an event listener you have to take care about changes in the original event listener and adjust your event listener accordingly

Or something like that.

Reply:
I added a `note` (right one?) for this, but rephrased it to what I believe you wanted to state. Please check if that was your intention to describe?

Reply:
Depends, "note" is okay, we have also "caution". Do as you think ;-)